lib/ab_test.rb

```diff
@@ -89,6 +124,11 @@ def active?
 
   private
 
+  def maxed?
```

Comment:
I'm wondering how we can _know_ that a test is maxed--is there a ticket to add that to the reporting emails or maybe create an alert or metric of some kind?

Reply:
That's a good thought. We could always resort to checking in a Rails console on a live instance, but I agree it'd be nice to have some indication in report mailers and/or logging. Especially for the report mailers, I think it's a good data point to be reporting for test progress.

Reply:
Updated in 7f32e19 to add participant counts / maximum to A/B test report mailer.

Can be previewed at the example mailer at http://localhost:3000/rails/mailers/report_mailer/ab_tests_report

![image](https://github.com/user-attachments/assets/39594988-e5ee-4e68-b361-82afa53b3cc5)